README.md

```diff
@@ -11,16 +11,39 @@ Image Template to build custom notebook images for JupyterHub.
 ## How to build the image
 
 ### Locally
+
+#### Prerequisites
+
+* **s2i**: We assume majority of users will be using linux and recommend using the the instructions
+[here](https://github.com/openshift/source-to-image#for-linux) to install s2i if needed.
+
+* **Podman**: Please follow the instructions [here](https://github.com/openshift/source-to-image#for-linux) for your
+linux distribution.
+
+* **Docker**: If you are using Fedora please use these [instructions](https://docs.docker.com/engine/install/fedora/)
+to install docker. If you are on RHEL8 you can run into issues with docker, please follow the additional documentation
+that can be found [here](docs/RHEL8_docker_install.md).
+
+#### Build
+
   You can build the image locally using s2i, by executing the following command
   in the root of your repository. <br>
-    `s2i build . quay.io/thoth-station/s2i-custom-notebook:latest MY-CUSTOM-IMAGE:latest`
+````
+$ s2i build . quay.io/thoth-station/s2i-custom-notebook:latest MY-CUSTOM-IMAGE:latest
```

Comment:
While you are making changes, can you please add that you can directly use the git repo url to build the image?
similar to what I wrote [here](https://github.com/AICoE/s2i-custom-notebook/tree/4n4nd-update-readme#locally)?
if not, I can just do it later

Reply:
sure : )

Reply:
thanks!